research/blueprint_algo/blueprint_short_deck_poker.py

```diff
@@ -283,16 +341,20 @@ def new_game(n_players: int) -> ShortDeckPokerState:
             else:
                 cfr(state, i, t)
         if t < LCFR_threshold & t % discount_interval == 0:
+            # TODO(fedden): Is discount_interval actually set/managed in
```

Comment:
Oh yes, the minutes I just converted to iterations for ease of mocking up. Might have forgotten to mention